docs/source/index.rst

```diff
@@ -15,7 +15,7 @@ Contents:
    Loaders
    LoggingSystem
    Sysinfo
-   Mux
+   TestParameters
```

Comment:
I have to say that the documentation changes make sense, but since this area has changed to much recently, it needs a more thorough review.  Please don't consider this brief doc review and endorsement of the overall status of the docs.

Since there are many earlier comments you can address, I'll try to do a proper doc review after this initial round.